include/global.h

```diff
@@ -1057,7 +1057,7 @@ struct SaveBlock1
     /*0x31A8*/ u8 giftRibbons[GIFT_RIBBONS_COUNT];
     /*0x31B3*/ struct ExternalEventData externalEventData;
     /*0x31C7*/ struct ExternalEventFlags externalEventFlags;
-    /*0x31DC*/ struct Roamer roamer;
+    /*0x31DC*/ struct Roamer roamer[ROAMER_COUNT];
```

Comment:
Isn't this going to break since we don't have the space in saveblock1 anymore?

Reply:
I can try to verify in-game if you’d like, but to my knowledge an array with a single element behaves the same as if it would just be the field by itself

Reply:
No, I mean the more roamers you add the more space will be needed but didn't we move all free space to saveblock3?

Reply:
SaveBlock1 still has 300 free bytes by default. I tried to up the limit until make complained and by default, there's space for 10 _concurrent_ roamers (which is way more than any vanilla game has ever had, and arguably more than good game design should allow), but this number can be upped by, moving the Roamer struct to SaveBlock3, by toggling settings of the SaveBlock cleansing branch, or by removing the filler at the end of the Roamer struct. Plenty of options.